apps/prairielearn/src/pages/instructorAssessmentOverrides/instructorAssessmentOverrides.js

```diff
@@ -0,0 +1,78 @@
+const error = require('@prairielearn/error');
```

Comment:
Even if you aren't prepared to author the file in TypeScript just yet, we should still opt in to type-checking.

```suggestion
// @ts-check
const error = require('@prairielearn/error');
```